src/Relude/String.hs

```diff
@@ -10,6 +10,33 @@ Stability:   Stable
 Portability: Portable
 
 Type classes for conversion between different string representations.
+
+The table below represents the general idea of conversion between the following types:
+
+* 'Text'
+* 'String'
+* 'ByteString'
+* 'LText'
+* 'LByteString'
+* 'ShortByteString'
+
+
++-------------------+---------------------+--------------+--------------+--------------------+------------------------+--------------------------+
+|     From \\ To    |       'Text'        |   'String'   | 'ByteString' |      'LText'       | 'LByteString'          | 'ShortByteString'        |
++===================+=====================+==============+==============+====================+========================+==========================+
+| 'Text'            | ~                   | 'toString'   | 'encodeUtf8' | 'toLazy'/'toLText' | 'encodeUtf8'           | 'encodeUtf8'             |
```

Comment:
I think it would be nice to make types in the first column bold as well, to make the table prettier and to hint a clearer difference between types and functions